cmd/cloud-controller-manager/app/apis/config/types.go

```diff
@@ -23,10 +23,12 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// CloudControllerManagerConfiguration contains elements describing cloud-controller manager.
 type CloudControllerManagerConfiguration struct {
 	metav1.TypeMeta
 
-	// Generic holds configuration for a generic controller-manager
+	// GenericControllerManagerConfiguration holds configuration for GenericComponent
+	// related features both in cloud controller manager and kube-controller manager.
```

Comment:
The description should keep orignal, not change

Reply:
fixed